src/graphql/schema.gql

```diff
@@ -207,6 +207,7 @@ type Space {
   plugins: Any
   voting: SpaceVoting
   categories: [String]
+  validation: Strategy
```

Comment:
We should add a default value on formatSpace otherwise it will break cuz some space dont have this field

Reply:
When I checked the code on snapshot, formatSpace already has a basic Validation set if validation is undefined/ null @bonustrack